tests/pc/21_all_opt+beh+scr_pc.h

```diff
@@ -0,0 +1,193 @@
+/*
+ * c't-Bot
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General
+ * Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ * This program is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE. See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the Free
+ * Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ */
+
+#ifndef INCLUDE_BOT_LOCAL_OVERRIDE_H_
+#define INCLUDE_BOT_LOCAL_OVERRIDE_H_
+
+/* == bot-local.h == */
+
+/* System-Konfiguration --> Diese Einstellungen sind von der lokalen Hardware abhaengig, Veraenderungen sind nur bei Hardware-Umbauten noetig */
+//#define F_CPU	20000000UL			/**< CPU-Frequenz [Hz] (16000000UL oder 20000000UL) */
+#define UART_BAUD	115200		/**< Baudrate fuer UART-Kommunikation (moegliche Werte sind 57600, 115200, 230400, 500000) */
+#define ENABLE_RX0_PULLUP			/**< Aktiviert den internen Pullup fuer die RX-Leitung. Nicht aktivieren, falls entsprechender Hardware-Mod eingebaut ist! */
+#define EXPANSION_BOARD_AVAILABLE	/**< Erweiterungsmodul (MMC / WiPort) installiert */
+#define SPI_AVAILABLE				/**< verwendet den Hardware-SPI-Modus des Controllers, um mit der MMC zu kommunizieren. Muss ausserdem _immer_ an sein, wenn der Hardware-SPI-Umbau durchgefuehrt wurde! Hinweise in mcu/mmc.c beachten! */
+#define SPI_SPEED	2				/**< SPI-Clockfrequenz (falls SPI_AVAILABLE) als Teiler von F_CPU (moegliche Werte sind 2, 4, 8, 16, 32, 64, 128) */
+#define AUTO_DISPLAYLIGHT			/**< helligkeitsgesteuerte Displaybeleuchtung */
+#define DISTSENS_TYPE_GP2Y0A60		/**< Distanzsensor Typ GP2Y0A60 */
+
+/* I/O-Schnittstellen fuer Raspberry Pi */
+#define USB_UART_LINUX		/**< Wird ein USB UART (z.B. USB-2-Bot) Adapter fuer die Verbinung zum ATmega verwendet? */
+#define BOT_RESET_GPIO		"/sys/class/gpio/gpio17/value"	/**< Pfad zum Reset-GPIO vom ARM-Linux-Board (default: BCM-GPIO 17 auf Pin 11 der RPi-GPIO-Leiste) */
+#define ARM_LINUX_DISPLAY	"/dev/tty1"	/**< Konsole fuer Display-Ausgaben auf ARM-Linux-Board. "stdout" fuer Ausgabe auf stdout */
+
+
+/* == ct-Bot.h == */
+
+/* Logging-Funktionen */
+#define LOG_CTSIM_AVAILABLE					/**< Logging zum ct-Sim (PC und MCU) */
```

Comment:
Das geht nicht, diese Optionen schließen sich gegenseitig aus.

Reply:
per
`#ifdef LOG_UART_AVAILABLE`
`#define UART_AVAILABLE /**< Serielle Kommunikation */`
`#undef LOG_CTSIM_AVAILABLE`
...
`#endif`

ist das korrekt, aber genau aus diesem Grund hatte ich das hier so definiert.
Wie bereits erwähnt gehört das ohnehin eher in einen Testzweig. Das war in der Tat etwas unglücklich von mir das mit hier in den PR reinzustecken.